GUI/Configuration.cs

```diff
@@ -43,13 +43,13 @@ public class Configuration
         /// <param name="vis">true if visible, false if hidden</param>
         public void SetColumnVisible(string name, bool vis)
```

Comment:
Also, just saw that now, might `SetColumnVisibility()` be a better name? Because this function also sets columns to "invisible".

Reply:
Done.